### PR TITLE
Setup Travis config

### PR DESCRIPTION
**Pull request by pawamoy**
Add a configuration file for Travis continuous integration service (.travis.yml).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+language: python
+python: '3.5'
+sudo: false
+env:
+  - TOXENV=py26
+  - TOXENV=py27
```
Comment (RonnyPfannschmidt) [inline .travis.yml]: why use python 3.5 as the python and then only test on 2.7 and the completely dead 2.6

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - uname -a
+  - lsb_release -a
+
+install:
+  - pip install tox
+  - virtualenv --version
+  - easy_install --version
+  - pip --version
+  - tox --version
+branches:
```
Comment (RonnyPfannschmidt) [inline .travis.yml]: this severely limits experimentation in case of need

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  only:
+    - master
+script:
+  - tox -v
+after_failure:
```
Comment (RonnyPfannschmidt) [inline .travis.yml]: interesting approach

File: .travis.yml (hunk 1 above)
Comment (pawamoy) [reply .travis.yml]: I didn't want to add too much and kept what was already tested. I agree that python 2.6 is a bit old now and that detox should be tested against python 3.4, 3.5 and even 3.6. If all maintainers agree on this I can add these versions to be tested.

File: .travis.yml (hunk 3 above)
Comment (pawamoy) [reply .travis.yml]: Just took what I have in my own setups (coming from cookiecutter-pylibrary)

**Comment by hpk42** [comment]
i am fine with just py27 and py35,py36, if there are no problems then also py34.

**Comment by RonnyPfannschmidt** [comment]
btw, jsut wondering, why not use the travis python matrix and always use the py toxenv?

**Comment by pawamoy** [comment]
Aaaaaah I was wondering how to do that, thanks, I'll try. When it work maybe I can rebase to clean up...

**Comment by RonnyPfannschmidt** [comment]
correct - tox will always run the python version it was started with for the normal default testenv

so its not necessary to declare them at all

**Comment by pawamoy** [comment]
Alright, I squashed the commits. The Travis build will still fail without https://github.com/tox-dev/detox/pull/2 since (I guess) Travis builds use only one CPU and fail with previous timeout of 30 seconds.

See the build success on my fork combining the two PRs: https://travis-ci.org/Pawamoy/detox/builds/202230155

**Comment by obestwalter** [comment]
@Pawamoy thanks for the PRs.

One request for your future PRs (hopefully more to come :)) - please don't squash them before we merge. We can do this when merging if necessary/wanted. One disadvantage of squashing them beforehand is that the commit history might help understanding the "evolution" of a PR which is lost, when you sqash. The bigger point though, is that the links to the commits from review comments get lost (see above comments from Ronny that are pointing to nowhere now).

**Comment by obestwalter** [comment]
First build was successfull. Thanks again!

**Comment by pawamoy** [comment]
Yes, I saw afterwards that GitHub provides the ability to squash if needed. Won't do that again, thanks!

**Pull request merged**